pgx-pg-sys/src/submodules/guard.rs

```diff
@@ -76,17 +76,6 @@ fn take_panic_location() -> PanicLocation {
 thread_local! { pub(crate) static IS_MAIN_THREAD: once_cell::sync::OnceCell<()> = once_cell::sync::OnceCell::new() }
```

Comment:
Seems we should remove this too.

Reply:
I was slightly more hesitant to remove all of that machinery because it was alive for "not just in debug", so I couldn't immediately tell if it was important or not that it worked this way for the non-debug cases. But if you are confident that it is okay to remove, I will happily do so!